cpp.md

```diff
@@ -168,6 +168,28 @@ default:
  //code to be executed when all the above cases are not matched;    
 } 
 ```
+## Break and Continue Statements
+
+### 1. Break
+```c
+for(i=0; i<5; i++){
+   if(i==4){
```

Comment:
can you add more realistic example

Reply:
for break statement should i use an example of switch statement? that seems to be a more realistic one